tenders/common/elf.h

```diff
@@ -60,6 +60,11 @@ void elf_load(int bin_fd, const char *bin_name, uint8_t *mem, size_t mem_size,
  * will cause the executable to be rejected. (bin_name) is the file name of the
  * binary and is used to report errors.
  *
+ * XXX: In order to not have to deal with internal alignment issues and
+ * simplify parsing, this function supports PT_NOTE headers containing a
+ * *single* NOTE descriptor. Refer to the Solo5 linker scripts for how to
+ * produce ELF binaries using this scheme.
```

Comment:
Unfortunately not, but at this stage I expect those seeing these errors to be low-level folks dealing with libOS toolchain issues, so they'll have to manage. I will open an issue about clearer error reporting in the ELF loader, so we won't forget.

Reply:
As long as there is *an* error I think that's fine for now, I was mostly worried about the scenario where something gets silently ignored.

Reply:
Well, it'll error out complaining that it can't find an MFT1 or ABI1 note, whichever of them would end up getting packed 2nd in the PT_NOTE header. So, not terribly helpful but it is _an_ error :-)